stock_return_request/report/stock_return_report.xml

```diff
@@ -0,0 +1,141 @@
+        <t t-call="web.html_container">
+            <t t-call="web.external_layout">
+                <div class="page">
+                    <div class="row">
+                        <div class="col-xs-5 col-xs-offset-1">
+                            <div
+                                t-if="request.return_type == 'supplier' and request.partner_id"
+                            >
+                                <span><strong>Supplier Address:</strong></span>
+                            </div>
+                            <div
+                                t-if="request.return_type == 'customer' and request.partner_id"
+                            >
+                                <span><strong>Customer Address:</strong></span>
+                            </div>
+                            <div t-if="request.partner_id" name="partner_header">
+                                <div
+                                    t-field="request.partner_id"
+                                    t-options='{"widget": "contact", "fields": ["name", "phone"], "no_marker": True, "phone_icons": True}'
+                                />
+                                <p t-if="request.sudo().partner_id.vat"><t
+                                        t-esc="res_company.country_id.vat_label or 'TIN'"
+                                    />: <span
+                                        t-field="request.sudo().partner_id.vat"
+                                    /></p>
+                            </div>
+                        </div>
+                    </div>
+                    <br />
+                    <h1 t-field="request.name" class="mt0 pull-left" />
+                    <div class="clearfix" />
+                    <br />
+                    <h4>[<t t-esc="request.return_from_location.name" /> &#8594; <t
+                            t-esc="request.return_to_location.name"
+                        />]</h4>
+                    <t
+                        t-call="stock_return_request.stock_move_lines_to_return"
+                        name="move_lines"
+                    >
+                        <t
+                            t-set="move_lines"
+                            t-value="request.returned_picking_ids.mapped('move_line_ids')"
+                        />
+                        <t t-set="request_type" t-value="request.return_type" />
+                    </t>
+                    <br />
+                    <p t-esc="request.note" />
+                </div>
+            </t>
+         </t>
+    </template>
+
+    <template id="report_stock_return_request">
+        <t t-foreach="docs" t-as="request">
+            <t t-call="stock_return_request.report_stock_return_request_document" />
+        </t>
+    </template>
+
+    <record id="action_report_stock_return_request" model="ir.actions.report">
+        <field name="name">Stock Return Request Report</field>
+        <field name="model">report.stock.card.report</field>
```

Comment:
There is a dependency here on report.stock.card.report but this is not in the depends?